order-book/src/test/java/com/market/banica/order/book/model/ItemMarketTest.java

```diff
@@ -0,0 +1,85 @@
+package com.market.banica.order.book.model;
+
+import com.market.Origin;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ItemMarketTest {
+    private static final String EGGS_ITEM_NAME = "eggs";
+    private static final String RICE_ITEM_NAME = "rice";
+    private static final String MEAT_ITEM_NAME = "meat";
+
+    private final ItemMarket itemMarket = new ItemMarket();
+    private final Map<String, TreeSet<Item>> allItems = new ConcurrentHashMap<>();
+
+    @BeforeEach
+    public void setUp() {
+        TreeSet<Item> items = new TreeSet<>();
```

Comment:
it would be more clear to populate the items in a separate method:
TreeSet<Item> populateItems() {}

Reply:
Fixed